dex/src/main/scala/com/wavesplatform/dex/actors/orderbook/OrderBookActor.scala

```diff
@@ -127,20 +127,25 @@ class OrderBookActor(
     // DEX-1192 docs/places-and-cancels.md
     case request: ValidatedCommandWithMeta =>
       actualizeRules(request.offset)
-      lastProcessedOffset match {
-        case Some(lastProcessed) if request.offset <= lastProcessed => // Already processed
```

Comment:
I'd rather leave this place unchanged, because we will load a snapshot prior an order book deletion and process only `DeleteOrderBook`. The logic is same, but with less code.